packages/server/src/api/app/bsky/getBadgeMembers.ts

```diff
@@ -6,31 +6,40 @@ import { paginate } from '../../../db/util'
 export default function (server: Server) {
   server.app.bsky.getBadgeMembers(
     async (params: GetBadgeMembers.QueryParams, _input, _req, res) => {
-      const { uri, cid, limit, before } = params
+      const { uri, limit, before } = params
       const { db } = locals.get(res)
       const { ref } = db.db.dynamic
 
       let builder = db.db
-        .selectFrom('app_bsky_accepted_badge as accepted')
-        .where('accepted.subject', '=', 'uri')
-        .innerJoin('user', 'accepted.creator', 'user.did')
+        .selectFrom('app_bsky_badge as badge')
+        .innerJoin(
+          'app_bsky_badge_offer as offer',
+          'offer.badgeUri',
+          'badge.uri',
+        )
+        .innerJoin(
+          'app_bsky_badge_accept as accept',
+          'accept.badgeUri',
+          'badge.uri',
+        )
+        .innerJoin('user', 'user.did', 'accept.creator')
         .leftJoin('app_bsky_profile as profile', 'profile.creator', 'user.did')
+        .where('badge.uri', '=', uri)
+        .whereRef('offer.creator', '=', 'badge.creator')
+        .whereRef('offer.subject', '=', 'accept.creator')
+        .whereRef('accept.offerUri', '=', 'offer.uri')
         .select([
           'user.did as did',
           'user.username as name',
           'profile.displayName as displayName',
-          'accepted.createdAt as createdAt',
-          'accepted.indexedAt as indexedAt',
+          'offer.createdAt as offeredAt',
+          'accept.createdAt as acceptedAt',
         ])
 
-      if (cid) {
-        builder = builder.where('accepted.subjectCid', '=', cid)
-      }
```

Comment:
Any special reason this was nixed?

Reply:
honestly it was getting in the way & we didn't really need it 😅 (since badges shouldn't need to be edited)
but i can add it back in now to match other routes 👌

Reply:
Got it!